src/Altinn.Authorization.ServiceDefaults/src/ServiceDefaults/AltinnServiceDefaultOptions.cs

```diff
@@ -0,0 +1,49 @@
+namespace Altinn.Authorization.ServiceDefaults;
+
+/// <summary>
+/// Provides configuration options for setting up default Altinn services.
+/// </summary>
+public class AltinnServiceDefaultOptions
+{
+    /// <summary>
+    /// Configures which default services should be enabled during Altinn service setup.
+    /// All services are enabled by default.
+    /// </summary>
+    /// <param name="configureAddServices">An action to configure options for enabling or disabling specific services.</param>
+    /// <returns>The current <see cref="AltinnServiceDefaultOptions"/> instance.</returns>
+    public AltinnServiceDefaultOptions ConfigureEnabledServices(Action<EnabledServicesOptions> configureAddServices)
```

Comment:
Move this to an extension class. This also allows for creating extensions such as `AltinnServiceDefaultOptions.DisableApplicationInsights()`.